keystore/src/connection/platform/generic/meta_migrations/v16.rs

```diff
@@ -0,0 +1,251 @@
+use std::io::{Read, Write};
+
+use rusqlite::Transaction;
+
+use crate::{
+    CryptoKeystoreResult, MissingKeyErrorKind,
+    connection::{DatabaseConnection, KeystoreDatabaseConnection, TransactionWrapper},
+    entities::{Entity, EntityBase, EntityFindParams, EntityTransactionExt, StoredCredential, StringEntityId},
+    migrations::{StoredSignatureKeypair, V5Credential, migrate_to_new_credential},
+};
+
+pub(crate) const VERSION: u16 = 16;
+
+pub(crate) fn meta_migration(conn: &mut rusqlite::Connection) -> CryptoKeystoreResult<()> {
+    let tx = conn.transaction()?;
+    let mut stmt = tx.prepare(&format!(
+        "SELECT
+            id,
+            credential,
+            unixepoch(created_at) AS created_at,
+            signature_scheme,
+            pk,
+            keypair,
+            credential_id
+         FROM {credential_table}, {keypair_table}
+         WHERE {keypair_table}.credential_id = {credential_table}.id",
```

Comment:
I thought we determined that this is not actually the credential ID, but the client ID?